docs/reference/analysis/tokenfilters/apostrophe-tokenfilter.asciidoc

```diff
@@ -1,5 +1,84 @@
 [[analysis-apostrophe-tokenfilter]]
-=== Apostrophe Token Filter
+=== Apostrophe token filter
+++++
+<titleabbrev>Apostrophe</titleabbrev>
+++++
 
-The `apostrophe` token filter strips all characters after an apostrophe,
-including the apostrophe itself.
+Strips all characters after an apostrophe, including the apostrophe itself.
+
+[[analysis-apostrophe-tokenfilter-analyze-ex]]
+==== Example
+
+The following <<indices-analyze,analyze API>> request demonstrates how the
+apostrophe token filter works.
+
+[source,console]
+--------------------------------------------------
+GET /_analyze
+{
+  "tokenizer" : "standard",
+  "filter" : ["apostrophe"],
+  "text" : "Istanbul'a veya Istanbul'dan"
+}
+--------------------------------------------------
+
+The `apostrophe` token filter produces the following tokens:
+
+[source,text]
+---------------------------
+[ Istanbul, veya, Istanbul ]
+---------------------------
+
+/////////////////////
+[source,console-result]
```

Comment:
I would love any feedback here.

The token example is based on the one from the simple analyzer:
https://www.elastic.co/guide/en/elasticsearch/reference/master/analysis-simple-analyzer.html#_example_7

Reply:
From my understanding, this token filter was built to be used with Turkish. For clarity, it could be worth mentioning that explicitly, perhaps linking to the [Turkish analyzer](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-lang-analyzer.html#turkish-analyzer) (which makes use of this filter).

Reply:
That's a great idea. I've added this information to the description at the top with 04a33a69ca37a8d687a6c23e2882f2b8c08e0ddd.